tune-tikv-thread-performance.md

```diff
@@ -55,9 +61,22 @@ Starting from TiKV v5.0, all read requests use the unified thread pool for queri
 
 * The Raftstore thread pool.
 
-    The Raftstore thread pool is the most complex thread pool in TiKV. The default size (configured by `raftstore.store-pool-size`) is `2`. All write requests are written into RocksDB in the way of `fsync` from the Raftstore thread.
+    The Raftstore thread pool is the most complex thread pool in TiKV. The default size (configured by `raftstore.store-pool-size`) of this thread pool is `2`. For the StoreWriter thread pool, the default size (configured by `raftstore.store-io-pool-size`) of it is `0`.
+
+    - When the size of the StoreWriter thread pool is 0, all write requests are written into RocksDB in the way of `fsync` from the Raftstore thread. In this case, it is recommended to tune the performance as follows:
+
+        - Keep the overall CPU usage of the Raftstore thread below 60%. When the number of Raftstore thread is 2, the **TiKV-Details**、**Thread CPU**、**Raft store CPU** on Grafana are needed to keep within 120%. Due to I/O requests, Raftstore threads cannot reach 100% CPU usage theoretically.
+        - Do not increase the size of the Raftstore thread pool to improve write performance without consideration, because this might increase the disk burden and degrade performance.
+
+    - When the size of the StoreWriter thread pool is not 0, all write requests are written into RocksDB in the way of `fsync` from the StoreWriter thread. In this case, it is recommended to tune the performance as follows:
+
+        - Only enable the StoreWriter thread pool when overall CPU resources are sufficient. When the StoreWriter thread pool is enabled, keep the CPU usage of the StoreWriter thread and the Raftstore thread below 80%.
+
+          Compared with the case that the write requests are completed from the Raftstore thread, in theory, the write requests processed from the StoreWriter thread can significantly reduce write latency and tail latency of data read. However, when the write speed grows faster, the number of Raft logs increases accordingly. This can cause the CPU overhead to increase for the Raftstore threads, the Apply threads, and the gRPC threads. In this case, insufficient CPU resources might offset the optimization effect, and as a result, the write speed might become slower than before. Hence, if the CPU resources are not sufficient, it is not recommended to enable the StoreWriter thread. Since the Raftstore thread sends most of the I/O requests to the StoreWriter thread, the CPU usage of the Raftstore thread need to keep below 80%.
+
+    - In most cases, the size of the StoreWriter thread pool can be set to 1 or 2. This is because the size of the StoreWriter thread pool affects the number of Raft logs, so the value of it should not be too large. If the CPU usage is higher than 80%, you can consider increasing its size.
```

Comment:
```suggestion
    - In most cases, set the size of the StoreWriter thread pool to 1 or 2. This is because the size of the StoreWriter thread pool affects the number of Raft logs, so the value of the thread pool size should not be too large. If the CPU usage is higher than 80%, consider increasing the thread pool size.
```